src/test/util/setup_common.cpp

```diff
@@ -195,6 +195,16 @@ TestingSetup::TestingSetup(const std::string& chainName, const std::vector<const
                              true);
     assert(!rv.has_value());
 
+    auto rv2 = VerifyLoadedChainstate(
+        *Assert(m_node.chainman.get()),
```

Comment:
```suggestion
        *Assert(m_node.chainman),
```

I think this should also compile?